selfdrive/car/tests/test_models.py

```diff
@@ -238,9 +253,9 @@ def test_panda_safety_rx_valid(self):
         if t > 1e6:
           self.assertTrue(self.safety.addr_checks_valid())
 
-          # No need to check relay malfunction on disabled routes (relay closed) or for reasonable fingerprinting time
-          # TODO: detect when relay has flipped to properly check relay malfunction
-          if self.openpilot_enabled and t > 5e6:
+          # No need to check relay malfunction on disabled routes (relay closed),
+          # or before fingerprinting is done (1s of tolerance)
+          if self.openpilot_enabled and t / 1e4 > (self.elm_frame + 100):
```

Comment:
The reason we need this extra second of tolerance is because the safety mode went to silent after elm, before finally switching to the car safety mode and opening the relay. We can fix that, but we'd need two frame variables. Tolerance is fine for now.

Reply:
this should be a comment in the code, not the PR